gnomad_qc/v4/annotations/compute_coverage.py

```diff
@@ -312,13 +313,21 @@ def get_coverage_resources(
         },
         pipeline_input_steps=[compute_coverage_ht],
     )
+    export_an_tsv = PipelineStepResourceCollection(
+        "--export-all-sites-an-release-tsv",
+        output_resources={
+            "allele_number_tsv": release_all_sites_an_tsv_path(data_type, test=test)
```

Comment:
Wont the release version always be the `CURRENT_ALL_SITES_AN_VERSION["exomes"]`. I know they are in sync now but should probably adjust the function call or function itself?

Reply:
yup, thanks for catching. I fixed this for the other functions in my combined freq PR, but forgot to change it here